cypress/e2e/cloud/billing.test.ts

```diff
@@ -67,6 +67,16 @@ describe('Billing Page PAYG Users', () => {
             }).then(() => {
               cy.visit(`/orgs/${id}/billing`)
               cy.getByTestID('billing-page--header').should('be.visible')
+
+              // note:  when the 'multiAccount' feature flag is on/removed (so always on)
+              // remove the above line, replace with:
+
+              // cy.getByTestID('accounts-billing-tab').should('be.visible')
+              //
+              // cy.getByTestID('accounts-billing-tab').should(
+              //     'have.class',
+              //     'cf-tabs--tab__active'
+              // )
```

Comment:
I think instead of having comments here, it would be much more useful to add an issue to add an e2e test for the new functionality, which is the system we have to track future work. It makes the work needed visible to people who aren't in this code, like managers and directors and VPs.

Adding commented out code to be removed at some time in the future can lead to a lot of confusion and has a tendency to add tech debt without much benefit. If the system changes, or what needs to happen changes, this comment will be wrong, but it's unlikely it'll be updated.

Reply:
i tried adding e2e tests; and then they got removed because the feature flag setting is flaky.  
i'm worried that the issue will become lost, but if you feel strongly i can create an issue with this text in it.

Reply:
@hoorayimhelping  and I discussed on slack; he does feel strongly, so in a new PR will be removing the comments, and put in an issue here: https://github.com/influxdata/ui/issues/3651